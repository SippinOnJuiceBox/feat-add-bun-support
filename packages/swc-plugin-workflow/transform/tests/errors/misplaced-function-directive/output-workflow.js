/**__internal_workflows{"steps":{"input.js":{"badStep":{"stepId":"step//input.js//badStep"}}}}*/;
export async function badStep() {
    return globalThis[Symbol.for("WORKFLOW_USE_STEP")]("step//input.js//badStep")();
}
export const badWorkflow = async ()=>{
    console.log('hello');
    // Error: directive must be at the top of function
    'use workflow';
    return true;
};
