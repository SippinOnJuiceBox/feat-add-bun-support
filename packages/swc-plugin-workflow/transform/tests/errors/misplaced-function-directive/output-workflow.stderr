  x The "use step" directive must be at the top of the function body
   ,-[input.js:4:1]
 3 |   // Error: directive must be at the top of function
 4 |   'use step';
   :   ^^^^^^^^^^^
 5 |   return x;
   `----
